structure/0039-pr-policy.md

```diff
@@ -0,0 +1,73 @@
+# Quilt PR Policy
+# Summary and Scope
+This RFC defines a system for merging Pull Requests (PR)s into Quilt projects. It applies to any repositories which opt-in to this system by adding a section (given below) to their CONTRIBUTING file.
+This RFC does not seek to require projects to use this process, but we encourage all projects expecting many pull requests to do so.
+
+# Motivation
+This RFC intends to make the process of creating, reviewing, and merging PRs easier for both Quilt teams and potential contributors.
+
+The goals for the PR process are:
+- **Democratic** -- Features are added through consensus of the team, and not through a benevolent dictator for life (BDFL) who has total control over the project.
+- **Quick** -- Tools should be in place to allow the team to shut down bikeshedding and say good enough on features if needed. However, this doesn't mean that pull requests should be easily forced through.
+- **Hands-off** -- Every team member shouldn't have to approve every PR. The team as a whole has a responsibility to ensure PRs are reviewed in a timely manner, but no person should have a responsibility to view every change.
+
+It is **not a goal** of this PR to be fool-proof from bad actors within the Quilt technical teams abusing the system. Being on any team in Quilt requires trust, and bad actors should be removed instead of worked around.
+
+# Explanation
+## Vocab
+- **Required Approvals**: The total number of reviews needed by the team assigned to a PR, or that team's superiors.
```

Comment:
An approval should be defined as using the review feature of github to make it more clear who is approving.